modules/examples/routeguide/server/src/main/scala/ServerAppIO.scala

```diff
@@ -21,6 +21,7 @@ import higherkindness.mu.rpc.server.{AddService, GrpcServer}
 import higherkindness.mu.rpc.config.server.BuildServerFromConfig
 import org.log4s._
 import example.routeguide.protocol.Protocols.RouteGuideService
+import example.routeguide.protocol.Protocols
```

Comment:
Do we need this import?

Reply:
I don't think so, I tried to import Protocols object because my IDE cannot resolve RouteGuideService, but surprisingly  it works anyway so I'll revert the change and let it as the master branch